argo-cd-apps/base/member/infra-deployments/backup/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- backup.yaml
```

In order to not hardcode the member cluster names, apply the following change in addition to the one in sibling backup.yaml file:
```suggestion
- backup.yaml
components:
  - ../../../../k-components/deploy-to-member-cluster-merge-generator

```